ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-main.yaml

```diff
@@ -7,23 +7,15 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-8-release-golang-1.17-openshift-4.10
+    tag: golang-1.21
```

Comment:
Can't say whether the change from the openshift based image can have some negative consequences, but if that works, I am fine with this change :slightly_smiling_face:

Reply:
This image is currently being used in the configurations for both [v1.9](https://github.com/openshift/release/blob/102a85a76f3d4d3c33d842f04b18cb0e6a1d6a66/ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-v1.9-branch.yaml#L10) and [v1.7](https://github.com/openshift/release/blob/102a85a76f3d4d3c33d842f04b18cb0e6a1d6a66/ci-operator/config/opendatahub-io/kubeflow/opendatahub-io-kubeflow-v1.7-branch.yaml#L10) configs, this golang image seems to be functioning as expected so far